packaging/google-guest-agent.spec

```diff
@@ -38,7 +38,7 @@ Contains the Google guest agent binary.
 %autosetup
 
 %build
-for bin in google_guest_agent google_metadata_script_runner gce_workload_cert_refresh; do
+for bin in google_authorized_keys_guest google_guest_agent google_metadata_script_runner gce_workload_cert_refresh; do
```

Comment:
ok, we can't rename this directory, or the windows builds will break. i think you should restore the name but add rename steps to the debian rules and rpm spec. or else we would want to coordinate with the windows releases to ensure they don't break with the rename.

Reply:
oh, and this PR is currently missing the debian updates

Reply:
By rename steps, would this be sufficient to do a similar renaming in this file (without changing the directory name)? 

install -p -m 0755 google_guest_agent/google_authorized_keys %{buildroot}%{_bindir}/google_authorized_keys_guest

Reply:
i think it should work; but you should be able to test it out yourself.

Reply:
tested and seems to work